tracer/test/snapshots/AwsSqsTests.NetCore.verified.txt

```diff
@@ -54,14 +73,124 @@
   {
     TraceId: Id_1,
     SpanId: Id_5,
+    Name: http.request,
+    Resource: POST localhost:00000/,
+    Service: Samples.AWS.SQS-http-client,
+    Type: http,
+    ParentId: Id_2,
+    Tags: {
+      component: HttpMessageHandler,
+      env: integration_tests,
+      http-client-handler-type: System.Net.Http.HttpClientHandler,
+      http.method: POST,
+      http.status_code: 200,
+      http.url: http://localhost:00000/,
```

Comment:
Interesting that these http-client spans are pretty useless because they don't include the URL path, means you can't identify them at all!

Reply:
Yep, I think the message contains all of the pertinent information so that's why it's super important that we're instrumenting the SDK!